spec/calendar.html

```diff
@@ -0,0 +1,153 @@
+      <li>has a *"length"* property whose value is 1.</li>
+    </ul>
+
+    <emu-clause id="sec-temporal.calendar">
+      <h1>Temporal.Calendar ( _id_ )</h1>
+      <p>When the `Temporal.Calendar` function is called, the following steps are taken:</p>
+      <emu-alg>
+        1. If NewTarget is *undefined*, then
+          1. Throw a *TypeError* exception.
+        1. Let _calendar_ be ? OrdinaryCreateFromConstructor(NewTarget, %Temporal.Calendar.prototype% &laquo; [[InitializedTemporalCalendar]], [[ID]] &raquo;).
+        1. Set _calendar_.[[ID]] to _id_.
+        1. Return _calendar_.
+      </emu-alg>
+    </emu-clause>
+  </emu-clause>
+
+  <emu-clause id="sec-properties-of-the-temporal-calendar-constructor">
+    <h1>Properties of the Temporal.Calendar Constructor</h1>
+    <p>The Temporal.Calendar prototype:</p>
+    <ul>
+      <li>has a [[Prototype]] internal slot whose value is %FunctionPrototype%.</li>
+      <li>has the following properties:</li>
+    </ul>
+
+    <emu-clause id="sec-temporal-calendar-prototype">
+      <h1>Temporal.Calendar.prototype</h1>
+      <p>The initial value of Temporal.Calendar.prototype is %Temporal.Calendar.prototype%.</p>
+      <p>This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
+    </emu-clause>
+
+    <emu-clause id="sec-get-temporal-calendar-@@species">
+      <h1>get Temporal.Calendar [ @@species ]</h1>
+      <p>
+        `Temporal.Calendar[@@species]` is an accessor property whose set accessor function is *undefined*.
+        Its get accessor function performs the following steps:
+      </p>
+      <emu-alg>
+        1. Return the *this* value.
+      </emu-alg>
+
+      <p>The value of the *"name"* property of this function is *"get [Symbol.species]"*.</p>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal.calendar.from">
+      <h1>Temporal.Calendar.from ( _item_ )</h1>
+      <p>The `from` method takes one argument: item. The following steps are taken:</p>
+      <emu-alg>
+        1. If _item_ has an [[InitializedTemporalCalendar]] internal slot, then
+          1. Return _item_.
+        1. If Type(_item_) is Object and _item_ is not *null*, then
+          1. Return _item_.
+        1. Let _stringIdent_ be ? ToString(_item_).
+        1. Return ? GetBuiltinCalendar(_stringIdent_).
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal-calendar-internal-slots">
+      <h1>Internal Slots</h1>
+
+      <p>The value of the [[BuiltinCalendars]] internal slot must be a record with keys corresponding to each supported calendar system and their corresponding values should be constructor objects that construct a Temporal.Calendar object corresponding to the calendar system.</p>
```

Comment:
Do you really want to use "should" in this sentence?

Reply:
Changing to "must".